pkg/cmd/pulumi/util.go

```diff
@@ -615,12 +619,16 @@ func isGitWorkTreeDirty(repoRoot string) (bool, error) {
 
 // getUpdateMetadata returns an UpdateMetadata object, with optional data about the environment
 // performing the update.
-func getUpdateMetadata(msg, root, execKind, execAgent string, updatePlan bool) (*backend.UpdateMetadata, error) {
+func getUpdateMetadata(
+	msg, root, execKind, execAgent string, updatePlan bool, cmd cobra.Command,
+) (*backend.UpdateMetadata, error) {
```

Comment:
This should really not return an error because it's not exported and there is no path in it that returns an error.